testing/acceptance_test.py

```diff
@@ -541,6 +542,28 @@ def test_foo(data):
         res = pytester.runpytest(p)
         res.assert_outcomes(passed=3)
 
+    @pytest.mark.filterwarnings("ignore:'encoding' argument not specified")
+    def test_command_line_args_from_file(
+        self, pytester: Pytester, tmp_path: Path
+    ) -> None:
+        pytester.makepyfile(
+            test_file="""
+            import pytest
+
+            class TestClass:
+                @pytest.mark.parametrize("a", ["x","y"])
+                def test_func(self, a):
+                    pass
+            """
+        )
+        tests = [
+            "test_file.py::TestClass::test_func[x]",
+            "test_file.py::TestClass::test_func[y]",
+        ]
+        args_file = pytester.maketxtfile(tests="\n".join(tests))
+        result = pytester.runpytest(f"@{args_file.absolute()}")
```

Comment:
I think it will actually be more realistic without the `absolute`:

```suggestion
        result = pytester.runpytest(f"@{args_file}")
```